### PR TITLE
Add audio_stream_out()

### PR DESCRIPTION
**Pull request by chanijjani**
This function just wraps up 'pcm_writei()" and "pcm_prepare()" as suggested.

### CODE REVIEW

**Comment by chanijjani** [comment]
I included 'tinyalsa/tinyalsa.h' in audio_manager layer to avoid build errors due to absense of references to 'struct pcm'.

**Comment by chanijjani** [comment]
https://android.googlesource.com/device/samsung/tuna/+/ics-mr1/audio/audio_hw.c

In this source, 'struct stream_in/out' include 'struct pcm *pcm' as a member variable.

File: framework/src/media/audio/audio_manager.c [hunk 1]
```diff
@@ -145,3 +145,16 @@ audio_manager_result_t get_avail_audio_device_id(unsigned int *device_id)
 	return AUDIO_MANAGER_SUCCESS;
 }
 
+audio_manager_result_t audio_stream_out(struct pcm *pcmOut, char* buffer, unsigned int num_read, unsigned int *frame)
```
Comment (Taejun-Kwon) [inline framework/src/media/audio/audio_manager.c]: As I reviewed before, pcm should be handled inside of audio manager...
This parameters means pcm should be handled in above layer.
Handling pcm for each device is reason why we making audio manager.
flow will be like a below.
1.MediaPlayer::create() will call set_streamout_device() -> find available device
2. MediaPlayer::prepare() will call set_ready_streamout_device() -> pcm_open(config  & pcm_prepare)
3. MediaPalyer::play() will call audio_stream_out(buffer) 
So, each device should handle its own pcm structure .

Comment (chanijjani) [reply framework/src/media/audio/audio_manager.c]: Okay, I'll modify!

**Pull request closed without merging**